packages/ui-link/src/Link/index.tsx

```diff
@@ -182,6 +182,23 @@ class Link extends Component<LinkProps, LinkState> {
     return hasVisibleChildren(this.props.children)
   }
 
+  get role() {
+    const { ariaRole, onClick } = this.props
+
+    // null will reset the role
+    if (ariaRole === null) {
+      return undefined
+    }
```

Comment:
hmm I dont like this logic, IMO we should treat `null` and `undefined` the same whenever possible (and we're doing it in all the places I know.). I'd have rather something like this:

- `undefined`/`null`/any other value: just pass it down
- `forceButtonRole`: if has onClick handler force button role, otherwise undefined

Reply:
Yeah you are right, I wasn't very happy with it either. I refactored it and added the prop that turns this logic off.